syne_tune/backend/scheduling.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
I think a better place for this would be `syne_tune/remote`, no?

Reply:
Good idea, moved.